src/controllers/controller.radar.js

```diff
@@ -21,12 +21,16 @@ defaults._set('radar', {
 	}
 });
 
-function nextItem(collection, index) {
-	return index >= collection.length - 1 ? collection[0] : collection[index + 1];
+function nextItem(collection, index, loop) {
+	return index >= collection.length - 1
```

Comment:
the nested ternary statements are a little hard to read. I might make it:
```
if (index >= collection.length - 1) {
  return loop ? collection[0] : collection[index];
}
return collection[index + 1];
```

Reply:
I just realized, `radar` does not support `monotone`. IMO This updating of control points should be done in `Line` element instead. That would again remove some duplication, but will PR that at a later time as this is big enough already.

Reply:
And done